docs/user/guides/access/terminal.md

```diff
@@ -1,4 +1,5 @@
 ---
+title: SSH or Terminal Access
```

Comment:
this is perfecto, but we should probably note somewhere that if using our hardware or installing via ISO the root user is not accessible people often are wondering why they cant and its built into the OS that root is only accessible via direct console access or elevating to root after logging in with other credentials.  but we need to revamp the installer flow and make the host credentials recoverable so idk this is a major sticking point and a bit ago pablo and I were talking about how to simplify this step we gotta brainstorm a bit. but merge took precedence.

Reply:
Haven't changed these docs yet, so we're gucci. Did only change the title. But agree, gotta streamline this.